src/applications/disability-benefits/996/pages/contactInformation.js

```diff
@@ -121,7 +70,6 @@ const contactInfo = {
           'country',
           'addressLine1',
           'addressLine2',
-          'addressLine3',
```

Comment:
Was this `addressLine3` not needed? Asking just out of general curiosity.

Reply:
tldr; all forms except 526 only have 2 lines; this code was originally copied from there.

The address line 3 is only needed for form 526 because it interacts with EVSS, and has an address line 1 limit of 20 characters. And I guess line 3 was added to make up for it? We have a ticket open trying to figure out how to better deal with it - https://github.com/department-of-veterans-affairs/va.gov-team/issues/5225 - but the 996 form (Higher-Level Review) doesn't need line 3. It was in there because I originally copied my code from form 526. And now, we have forwarding addresses disabled because the back-end isn't handling temporary forwarding addresses right now.